README.rst

```diff
@@ -41,7 +41,7 @@ use this package for volume detection.
 
 Just perform the following commands for a basic installation::
 
-    apt-get install python-setuptools xmount ewf-tools afflib-tools sleuthkit
+    apt-get install python-setuptools xmount ewf-tools afflib-tools sleuthkit qemu-nbd
```

Comment:
I feel this is not a requirement for a basic setup, as it is not a common disk image type to encounter.